### ISSUE TITLE
MyPy Error on 5.10.0

### ISSUE THREAD

**Issue by nagibyro** [omarryhan/aiogoogle#143]
Hello,

With the `5.10.0` release we run into a syntax error in `aiogoogle/resource.py:38` when running mypy on our project.

```
[~/projects/foo]$ poetry update aiogoogle
Updating dependencies
Resolving dependencies... (1.7s)

Package operations: 0 installs, 1 update, 0 removals

  • Updating aiogoogle (5.9.0 -> 5.10.0)

Writing lock file
[~/projects/foo]$ poetry run mypy --config-file ./pyproject.toml src/
/Users/username/Library/Caches/pypoetry/virtualenvs/foo-vYsNM9k8-py3.12/lib/python3.12/site-packages/aiogoogle/resource.py:38: error: invalid syntax  [syntax]
Found 1 error in 1 file (errors prevented further checking)
```

Our mypy config is simple

```toml
# pyproject.toml
[tool.mypy]
plugins = ["mypy_django_plugin.main"]
ignore_missing_imports = true
```

We downgraded to `5.9.0` and the issue was fixed

```
[~/projects/foo]$ poetry update aiogoogle
Updating dependencies
Resolving dependencies... (1.7s)

Package operations: 0 installs, 1 update, 0 removals

  • Updating aiogoogle (5.10.0 -> 5.9.0)


[~/projects/foo]$ poetry run mypy --config-file ./pyproject.toml src/
Success: no issues found in 137 source files
```

EDIT: additional context we are using mypy `1.10.1`

**Comment by omarryhan** [comment]
Thanks for a detailed error report @nagibyro!

Do you mind sharing what aiogoogle/resource.py:38 looks like on your end? I tried downloading the published package from the Python package repo and it seems line 38 is a comment. Very strange it's causing a syntax error.

Also, this is the 5.10.0 PR: https://github.com/omarryhan/aiogoogle/commit/1c78ecf1a23591e22149cf743cd363dc4a1b78c5, I doubt it's what caused it.

**Comment by nagibyro** [comment]
Confirmed on my end aiogoogle `resource.py` in our site-packages looks the same as @TomerYogev's

**Comment by nagibyro** [comment]
Simply changing the comment to not be `# type: ...` fixes the mypy issue, opened a pr just rewording.

**Comment by omarryhan** [comment]
Awesome! Thanks a lot @nagibyro !

Fix released in v5.11.0
